packages/components/src/lib/auth/app-authenticator.js

```diff
@@ -105,12 +105,14 @@ export default class AppAuthenticator {
           // Unrecoverable error handler
           this.auth0Lock.on('unrecoverable_error', (error) => {
             console.error('Auth0 Lock unrecoverable error: ', error)
+            // eslint-disable-next-line prefer-promise-reject-errors
```

Comment:
Do you really have this error (es-lint promise-reject-error) ?
I have fixed it in the previous PR with adding new Error.
Why did you add this exception?

Reply:
Oh, probably you've fixed it upstream, but I was working with the code based on the earlier version of master. So I've seen the error, suppressed it, and only after that merged your changes with the fixes. So I will remove those as they are no longer needed. Thanks for catching this!